README.md

```diff
@@ -2,14 +2,13 @@
 
 ## What? ###
 
-Kapta Mobile is a Progressive Web App enabling users to generate maps from WhatsApp chats.
+Kapta Mobile is a Progressive Web App to create WhatsApp Maps. 3 steps:
```

Comment:
I think the previous wording implying that the maps are generated from the whatsapp data makes it more obvious what we're doing
If you want to keep it short, maybe "create maps from Whatsapp in 3 steps:..."